tests/foreman/api/test_contentmanagement.py

```diff
@@ -1506,3 +1506,72 @@ def test_positive_sync_CV_to_multiple_LCEs(
 
         # Check all sync tasks finished without errors.
         self.wait_for_sync(module_capsule_configured)
+
+    @pytest.mark.tier4
+    @pytest.mark.skip_if_not_set('capsule')
+    def test_positive_capsule_sync_status_persists(
+        self, target_sat, module_capsule_configured, function_org, function_product, function_lce
+    ):
+        """Synchronize a capsule, delete the task in foreman-rake,
+        and then verify that the capsule is still listed as synced
+
+        :id: aa3d7dbb-8cbb-441c-85c2-70e85794c3a9
+
+        :steps:
+            1. Create and Sync a Repo
+            2. Associate lifecycle env with capsule
+            3. Create a CV with the repo
+            4. Publish CV
+            5. Promote to lifecycle env
+            6. Sync Capsule
+            7. Delete the task using foreman-rake console
+            8. Verify the status of capsule is still synced
+        """
+        repo_url = settings.repos.yum_1.url
+        repo = entities.Repository(product=function_product, url=repo_url).create()
```

Comment:
Switch to using sat object
```suggestion
        repo = target_sat.api.Repository(product=function_product, url=repo_url).create()
```